lib/govuk_navigation_helpers/taxon_breadcrumbs.rb

```diff
@@ -10,9 +10,11 @@ def breadcrumbs
       end
 
       ordered_parents << { title: "Home", url: "/" }
+      ordered_parents.reverse!
+      ordered_parents << { title: content_item.title, url: "#content" }
```

Comment:
Since this new breadcrumb isn't a parent, what do you think of keeping the `ordered_parents` as-is and below change the code to:

```
content_item_breadcrumb = { title: content_item.title, url: "#content" }

{
  breadcrumbs: ordered_parents + content_item_breadcrumb
}
```